### ISSUE TITLE
Better handle processes which exited with error

### ISSUE THREAD

**Issue by dkarlovi** [dpfaffenbauer/ProcessManager#54]
| Q                | A
| ---------------- | -----
| Bug report?      | no
| Feature request? | yes
| BC Break report? | no
| RFC?             | yes
| Branch?          | master

Currently, the processes which fail are not marked as such in the UI, they are just kept around and the impression (from users) is the process "hanged".

A better approach would be to keep track of the process execution and exit codes, marking the processes which exited unexpectedly in some way (maybe red-ish background or something?), signaling to the user this went badly.

**Comment by dpfaffenbauer** [comment]
yep, we currently don't monitor the process at all... so this would also allow us to monitor process which do not give directly feedback to the Process instance as well

**Comment by dkarlovi** [comment]
The first step would be to just change the status to `Failed`, even that would give more information to the person looking at it.
